README.md

```diff
@@ -90,14 +90,14 @@ This will kick off a build [in CircleCI](https://circleci.com/gh/mozilla/fxa-ema
      this instance will handle requests from.
      Valid values are `sendgrid` and `socketlabs`.
 
-   * `SQS_SUFFIX`:
```

Comment:
With the introduction of explict queue names, we didn't use `SQS_SUFFIX` anymore. Not sure how painful it is switching and deploying in stage/dev and if we should add it back.